operacje_na_listach/Function_to_sum.py

```diff
@@ -0,0 +1,8 @@
+def sum(list):
+    variable= 0
```

Comment:
maybe `total` or something like that instead of `variable`? `variable` is too general name